src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -31,19 +31,6 @@ public static ScriptSettingsManager Instance
         /// </summary>
         public virtual bool IsAppServiceEnvironment => !string.IsNullOrEmpty(GetSetting(EnvironmentSettingNames.AzureWebsiteInstanceId));
 
-        /// <summary>
-        /// Gets a value indicating whether we are running in a Linux container
-        /// </summary>
-        public virtual bool IsLinuxContainerEnvironment
```

Comment:
You mentioned the plan is for this class to go away and all helper properties to be extension methods off of environment/config? It looks like some properties have been migrated but the type is still used elsewhere?